frontend/app/components/progress.tsx

```diff
@@ -26,13 +26,17 @@ const indicatorBaseClassName = 'h-full w-full flex-1 transition-all';
 export interface ProgressProps extends React.ComponentPropsWithoutRef<typeof ProgressPrimitive.Root> {
   size?: keyof typeof sizes;
   variant?: keyof typeof variants;
+  label?: string;
```

Comment:
Set the label as required prop.

```suggestion
  label: string;
```